dev-tools/cmd/dashboards/export_dashboards.go

```diff
@@ -92,6 +92,13 @@ func Export(client *http.Client, conn string, dashboard string, out string) erro
 	}
 
 	data["objects"] = objects
+
+	// Create all missing directories
+	err = os.MkdirAll(path.Dir(out), os.ModePerm)
```

Comment:
`os.ModePerm` is too permissive. Use `0755` for dirs.

Reply:
changed